client/src/components/questions-answers/questionsList/QuestionListItem.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import AnswersPerQuestion from './AnswersPerQuestion';
+
+const QuestionListItem = props => {
+
+  let answers = props.question.answers;
+
+  return (
+    <>
+      <div>
+        <li className="QuestionItem">
+          Q: {props.question.question_body}
+        <br/>
+        </li>
+
+        {Object.keys(answers).map((keyName, i) =>
+          <AnswersPerQuestion
+          key={i}
+          answer={answers[keyName]}
+          />
+        )}
```

Comment:
```suggestion
const QuestionListItem =({ 
  question: { answers, question_body }
}) => {

  //let answers = props.question.answers; // props are always const because you don't mutate them

  return (
    <>
      <div>
        <li className="QuestionItem">
          Q: {question_body}
        <br/>
        </li>

        {Object.keys(answers).map((keyName, i) =>
          <AnswersPerQuestion
          key={i}
          answer={answers[keyName]}
          />
        )}
```